Makefile

```diff
@@ -53,6 +53,10 @@ new-repo:
 	$(MAKE) jobs
 	$(MAKE) prow-config
 
+validate-registry:
```

Comment:
Can this be `validate-step-registry` so folks don't confuse it with a container registry?

Reply:
/hold

in case you want this ;)